.github/workflows/tests.yml

```diff
@@ -229,6 +229,8 @@ jobs:
       - name: Build and push
         uses: docker/build-push-action@v6
         with:
+          cache-from: type=gha
```

Comment:
Interesting. Is this safe? Does it save a lot of time? Says here it is experimental, which makes me a little nervous :) https://docs.docker.com/build/ci/github-actions/cache/